package.json

```diff
@@ -15,9 +15,20 @@
     "npm": ">=2.0.0"
   },
   "scripts": {
-    "start": "grunt",
-    "test": "grunt test",
-    "postinstall": "bower install --config.interactive=false"
+    "update": "npm update && npm prune && bower install --allow-root --config.interactive=false && bower prune --allow-root --config.interactive=false",
+    "start": "gulp",
+    "start:prod": "gulp prod",
+    "start:debug": "gulp debug",
+    "lint": "gulp lint",
+    "test": "gulp test",
+    "test:server": "gulp test:server",
+    "test:client": "gulp test:client",
+    "test:e2e": "gulp test:e2e",
+    "test:watch": "gulp test:server:watch",
+    "test:watch:onlyChanged": "gulp test:server:watch --onlyChanged",
+    "prestart": "if [ ! -d node_modules ]; then npm install --quiet; fi",
```

Comment:
@simison Go ahead and remove this command. There's really no benefit to using this vs. npm install. Plus, this doesn't cover the edge case where a module was added or failed to install and npm install needs to be ran again even though node_modules directory exists.